dask/array/tests/test_linalg.py

```diff
@@ -449,35 +449,22 @@ def test_dask_svd_self_consistent(m, n):
         assert d_e.dtype == e.dtype
 
 
-@pytest.mark.slow
 def test_svd_compressed():
-    m, n = 2000, 250
-    r = 10
-    np.random.seed(4321)
-    mat1 = np.random.randn(m, r)
-    mat2 = np.random.randn(r, n)
-    mat = mat1.dot(mat2)
-    data = da.from_array(mat, chunks=(500, 50))
-
-    u, s, vt = svd_compressed(data, r, seed=4321, n_power_iter=2)
-
-    usvt = da.dot(u, da.dot(da.diag(s), vt))
-
-    tol = 0.2
-    assert_eq(
-        da.linalg.norm(usvt), np.linalg.norm(mat), rtol=tol, atol=tol
-    )  # average accuracy check
+    m, n = 100, 50
+    r = 5
+    a = da.random.random((m, n), chunks=(m, n))
 
-    u = u[:, :r]
-    s = s[:r]
-    vt = vt[:r, :]
+    # calculate approximation and true singular values
+    u, s, vt = svd_compressed(a, 2 * r, seed=4321, n_power_iter=0)  # worst case
+    s_true = scipy.linalg.svd(a.compute(), compute_uv=False)
 
-    s_exact = np.linalg.svd(mat)[1]
-    s_exact = s_exact[:r]
+    # compute the difference with original matrix
+    norm = scipy.linalg.norm((a - (u[:, :r] * s[:r]) @ vt[:r, :]).compute(), 2)
 
-    assert_eq(np.eye(r, r), da.dot(u.T, u))  # u must be orthonormal
```

Comment:
This should not be checked, if tsqr checks for orthogonality, and the normal svd (that is at the heart of tsqr 'option svd'), also does it. Then this statement is superfluous. samen for vt

Reply:
I would disagree that the underlying implementation of svd_compressed will always be based on tsqr, or that we shouldn't test the contract of the function instead of avoiding tests based on knowledge of how it currently works (which could certainly change).  I totally understand what you mean by it being superfluous now, but given the clarity of this check, how fast it is, and that it future proofs against simple mistakes, I think it should remain.

Reply:
I see that I have cut a corner here. Your point is valid, I will review this change tomorrow.